src/main/java/com/google/alpollo/ObjectifyWebFilter.java

```diff
@@ -0,0 +1,5 @@
+import javax.servlet.annotation.WebFilter;
+import com.googlecode.objectify.ObjectifyFilter;
+
+@WebFilter(urlPatterns = {"/*"})
+public class ObjectifyWebFilter extends ObjectifyFilter {}
```

Comment:
Maybe it is because I don't know Objectify, but I don't see how this is used. Maybe expand your comment to explain the role of this class in more details

Reply:
It is really unused. I can explain why I add it. It was because I tried to fix other errors and added everything that could help me. Now it isn't necessary.